### PR TITLE
feat: disk dashboard changes

### PR DESCRIPTION
**Pull request by Hardikl**
Grafana: http://10.249.25.52:3000/ admin/admin

### CODE REVIEW

File: conf/zapiperf/cdot/9.8.0/disk.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ plugins:
   # any names after the object names will be threated as
   # label names that will be added to instances
     - node
-    - aggr node
+    - aggr ...
```
Comment (cgrinds) [inline conf/zapiperf/cdot/9.8.0/disk.yaml:39]: is ... correct?

**Review by cgrinds — COMMENTED** [review]
nice job @Hardikl

File: conf/zapiperf/cdot/9.8.0/disk.yaml (hunk 1 above)
Comment (Hardikl) [reply conf/zapiperf/cdot/9.8.0/disk.yaml:39]: Yes. 
So we are aggregating data at node and aggr level and all metrics like user_read_chain and disk_busy would be available at node/aggr not at disk. Also talked with Vachgan and get to know that getting at disk level may impact performance as more calculations would be involved.

But we would need info of plex and raid where we can show how many plex,raid would be in system, so metric calculation wise it's same just with this aggr_xxx_xxx metric would be showing other labels as well along with node,cluster,datacenter.

Ex: for raid count:                    
count( count by(raid) (aggr_disk_busy{datacenter="$Datacenter",cluster="$Cluster",node=~"$Node"}) )

Comment (Hardikl) [reply conf/zapiperf/cdot/9.8.0/disk.yaml:39]: just to add, flashcache and flashpool charts are pending as my cluster don't have them, doing in next PR.

Comment (cgrinds) [reply conf/zapiperf/cdot/9.8.0/disk.yaml:39]: thanks - got it, so you're wanting to copy all labels of the original instance

**Review by cgrinds — APPROVED** [review]

**Pull request merged**